labgrid/remote/coordinator.py

```diff
@@ -110,6 +110,8 @@ def get_resources(self):
 
 @attr.s(eq=False)
 class ClientSession(RemoteSession):
+    session = attr.ib()
```

Comment:
Maybe rename do `name` or `id`?

Reply:
Actually, it looks like `RemoteSession` has a `name` attribute, so we might have to go for `id`.